tester_app/client/app/bundles/HelloWorld/startup/clientGlobals.jsx

```diff
@@ -0,0 +1,3 @@
+import helloWorldClientEntryComponent from './helloWorldClientEntryComponent';
+
```

Comment:
Let's call this 

```
startup/HelloWorldApp
containers/HelloWorld
components/HelloWorldWidget
```

We should update the readme on this naming...

@alexfedoseev @robwise @jbhatab @rstudner How do you like this convention?